src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -645,7 +650,7 @@ class LicenceService(
       }
 
       val updatedAdditionalConditionUploadSummary = condition.additionalConditionUploadSummary.map {
-        var uploadDetail = additionalConditionUploadDetailRepository.getById(it.uploadDetailId)
+        var uploadDetail = additionalConditionUploadDetailRepository.getReferenceById(it.uploadDetailId)
```

Comment:
Is this intentional? it doesn't seem related to your change and there's no associated updated tests?

Reply:
Is this in relation to a deprecated warning?

Reply:
yes, due to deprecation warning